### ISSUE TITLE
DWOC Contribution

### ISSUE THREAD

**Issue by VedantKhairnar** [VedantKhairnar/Cheat-Sheets#29]
Contributor for DWOC has to add atleast 5 cheatsheets and one person can contribute only once

**Comment by Amit0412** [comment]
HELLO VEDANT I WOULD LIKE TO CONTRIBUTE INTO THIS. SO CAN I GET THE OPPORTUNITY TO CONTRIBUTE IN THIS?THANKYOU

**Comment by Amit0412** [comment]
I HAVE 5+ CHEATSHEETS.

**Comment by Raghavi94** [comment]
I am interested in this issue..I will add cheatsheets related to python and web development.

**Comment by smriti1313** [comment]
Hi, I would like to contribute in this too. I can do python,html,css,javascript and data mining

**Comment by 07yuvraj** [comment]
hey i would like to add cheatcheets for web development(html,css,js),machine learning and c++(data structures).May i go ahead with this issue?

**Comment by VedantKhairnar** [comment]
Sure, You all can make your own issue with the list of all those CheatSheets you are adding:

1. Cheat Sheet 1
2. Cheat Sheet 2
3. Cheat Sheet 3
4. Cheat Sheet 4
5. Cheat Sheet 5

**Comment by Amit0412** [comment]
> Sure, You all can make your own issue with the list of all those CheatSheets you are adding:
> 
> 1. Cheat Sheet 1
> 2. Cheat Sheet 2
> 3. Cheat Sheet 3
> 4. Cheat Sheet 4
> 5. Cheat Sheet 5

ACTUALLY I JUST WANT TO ASK ONE THING THAT IN ORDER TO CREATE NEW ISSUE AS YOU SAID SO  WE HAVE TO CLICK ON NEW ISSUE BUTTON AND ADD THE CHEATSHEETS WRITE?DO WE HAVE TO FORK OR UPDATE SOMETHING?

**Comment by arnabsen1729** [comment]
@Amit0412 you don't have to use caps :smile:

**Comment by Pranjal-2001** [comment]
I am interested

**Comment by Pranjal-2001** [comment]
I want to add cheat sheet for python and a few topics in calculus which are frequently taught in ML tutorials.

**Comment by VedantKhairnar** [comment]
> > Sure, You all can make your own issue with the list of all those CheatSheets you are adding:
> > 
> > 1. Cheat Sheet 1
> > 2. Cheat Sheet 2
> > 3. Cheat Sheet 3
> > 4. Cheat Sheet 4
> > 5. Cheat Sheet 5
> 
> ACTUALLY I JUST WANT TO ASK ONE THING THAT IN ORDER TO CREATE NEW ISSUE AS YOU SAID SO WE HAVE TO CLICK ON NEW ISSUE BUTTON AND ADD THE CHEATSHEETS WRITE?DO WE HAVE TO FORK OR UPDATE SOMETHING?

no just quote the cheatsheets which you will be adding. Then if I approve, you can fork and proceed

**Comment by VedantKhairnar** [comment]
For contributing to DWOC,  list all those CheatSheets you are adding[Min 5]:

1) Cheat Sheet 1
2) Cheat Sheet 2
3) Cheat Sheet 3
4) Cheat Sheet 4
5) Cheat Sheet 5

Nt: Only 1 PR is accepted per person for DWOC

**Comment by chicken-biryani** [comment]
I have created a pull request that contains 5 images of ds and strings.

**Comment by VedantKhairnar** [comment]
Cool!
Merged it!

**Comment by himanshu007-creator** [comment]
sir, i just got a single cheat sheet regarding 30 days placement strategy. Please merge it under DWOC label.
https://github.com/VedantKhairnar/Cheat-Sheets/pull/26

**Comment by jsneha1710** [comment]
Sir, I have opened a new issue with 5 cheatsheets on Deep Learning, ML and Tensorflow, can you please check if it is ok and approve it.

**Comment by VedantKhairnar** [comment]
@himanshu007-creator cant do that. Goes against the rules. maybe you can find some more cheatsheets on google.
Sure @jsneha1710 will do
Thanks!!

**Comment by lokeshvenkatesan** [comment]
@VedantKhairnar I have added 5 cheatsheets, #116, kindly check it.

**Comment by VedantKhairnar** [comment]
Sure
